api/v1alpha1/api.go

```diff
@@ -212,6 +212,11 @@ type AIGatewayRouteRule struct {
 	// +optional
 	// +kubebuilder:validation:MaxItems=128
 	Matches []AIGatewayRouteRuleMatch `json:"matches,omitempty"`
+
+	// Timeouts defines the timeouts that can be configured for an HTTP request.
+	//
+	// +optional
+	Timeouts *gwapiv1.HTTPRouteTimeouts `json:"timeouts,omitempty"`
```

Comment:
Since the HTTPRoute resource is by backend, I think the timeout field is better be defined on the AIServiceBackend ? then you do not need to have the dedup logic.

Reply:
I will do this.